tutorials/pythongetstarted.md

```diff
@@ -0,0 +1,74 @@
+\page pythongetstarted Python Get Started
+
+Previous Tutorial: \ref cppgetstarted
+
+## Overview
+
+This tutorial describes how to get started using Ignition Math with Python.
+
+**NOTE**: If you have compiled Ignition Math from source, you should export
+your `PYTHONPATH`.
+
+```bash
+export PYTHONPATH=$PYTHONPATH:<path to your workspace>/install/lib/python
+```
+
+We will run through an example that determines the distance between two
+points in 3D space. Start by creating a bare-bones main file using the
+editor of your choice.
+
+```python
+def main():
+  pass
+
+if __name__ == "__main__":
+  main()
+```
+
+The easiest way to include Ignition Math is through `import ignition.math`.
+
+At this point your main file should look like
+
+```python
+import ignition.math
+
+def main():
+  pass
+
+if __name__ == "__main__":
+  main()
+```
+
+Now let's create to 3D points with arbitrary values. We will use the
+ignition.math.Vector3 class to represent these points. Ignition Math provides a handy
+ignition.math.Vector3d type which is a typedef of `Vector3d`. The result of this
```

Comment:
Does the `typedef` sentence make sense for Python users? Is `ignition.math.Vector3` something that they can use? I think this paragraph can be reworded for Python.